Sources/FioriARKit/ARCards/ViewModels/ARManager.swift

```
@@ -106,10 +106,10 @@ public class ARManager {
         
         if let worldConfig = configuration as? ARWorldTrackingConfiguration {
             worldConfig.detectionImages = self.referenceImages
-            self.configureSession(with: worldConfig)
+            do { try self.configureSession(with: worldConfig) } catch { print(error) }
```

is there a reason why we should swallow the error?